main.nf

```diff
@@ -42,12 +40,15 @@ log.info """\
         docker_container_registry: ${params.docker_container_registry}
         bgzip_extra_args = ${params.bgzip_extra_args}
         tabix_extra_args = ${params.tabix_extra_args}
-        multi_tumor_sample: ${params.multi_tumor_sample}
-        multi_normal_sample: ${params.multi_normal_sample}
-        tumor_only_mode: ${params.tumor_only_mode}
+
+    - sample names extracted from input BAM files and sanitized:
+        tumor_in: ${params.samples_to_process.findAll{ it.sample_type == 'tumor' }['orig_id']}
+        tumor_out: ${params.samples_to_process.findAll{ it.sample_type == 'tumor' }['id']}
+        normal_in: ${params.samples_to_process.findAll{ it.sample_type == 'normal' }['orig_id']}
+        normal_out: ${params.samples_to_process.findAll{ it.sample_type == 'normal' }['id']}
 """
 
-if (params.max_cpus < 16 || params.max_memory < 30) {
+if (params.max_cpus < 8 || params.max_memory < 16) {
```

Comment:
Should this be kept as is? If Mutect2 and MuSE require at least 16 CPUs and 32GB of memory, this would allow runs on systems with, for example, 9 CPUs and 17GB of memory

Reply:
I think it's useful - filling what otherwise is quite a large jump from F2 to F16.   I will more carefully adjust the values in `F8.config` tomorrow and maybe start a test run.  It will also require an update of `common_methods.config`.

Reply:
I agree it would be useful, if F8 will be acceptable, I would also look over the error messages with this particular check and make sure they're consistent.

On a more general note, I would suggest moving the addition of the F8.config to a separate PR as that will require more changes (to the submodule for example) and that change is technically unrelated to the intent of this PR